data/demo/data/modify-settings1.csv

```diff
@@ -1,3 +0,0 @@
-setting-1,setting-2
```

Comment:
if you remove this file and modify-settings2.csv then you will need to update where [this doc](https://github.com/MastodonC/witan.send/blob/master/doc/scenarios.md) refers to

Reply:
ok, I'll add these files in again with new setting IDs.

Reply:
I'm going to add these with a seperate PR @seb231